### ISSUE TITLE
[Request] Multiple new features

### ISSUE THREAD

**Issue by SimonFoxcoon** [ErrorFlynn/ytdlp-interface#97]
- AV1 support
- Premium/Enhanced Bitrate (possible by a custom argument)
- User created presets, for example one for archival (best quality everything, embed subs/thumb etc), one for more support for when quality doesn't matter that much (mp4/avc). It'd be faster than tweaking each setting to switch between them.
- Ability to disable Key moments from showing up when you select embed chapters. (Key moments are automatically generated chapters)

**Comment by SRC267** [comment]
Yep, looking forward for 1080p bitrate option. I know 4K is better when downsampled and usually grab them from YT. But there's a MV that only has 1080p Enhanced Bitrate with no 4K option. 

Here's the music video I'm wanting to in 1080p EB: https://www.youtube.com/watch?v=F1B9Fk_SgI0

**Comment by ErrorFlynn** [comment]
> Yep, looking forward for 1080p bitrate option. I know 4K is better when downsampled and usually grab them from YT. But there's a MV that only has 1080p Enhanced Bitrate with no 4K option.
> 
> Here's the music video I'm wanting to in 1080p EB: https://www.youtube.com/watch?v=F1B9Fk_SgI0

If you don't set a codec preference (and leave the container preference to "mp4" or "none"), then yt-dlp will automatically download format 616 (which has the enhanced bitrate), because it considers it the best. Otherwise it will ignore it, to meet your codec or `webm` requirement.

I will add an option to override the codec preference and download format 616 when it exists (thanks for providing a link to 1080p EB video, I use it for testing).

**Comment by SRC267** [comment]
> > Yep, looking forward for 1080p bitrate option. I know 4K is better when downsampled and usually grab them from YT. But there's a MV that only has 1080p Enhanced Bitrate with no 4K option.
> > Here's the music video I'm wanting to in 1080p EB: https://www.youtube.com/watch?v=F1B9Fk_SgI0
> 
> If you don't set a codec preference (and leave the container preference to "mp4" or "none"), then yt-dlp will automatically download format 616 (which has the enhanced bitrate), because it considers it the best. Otherwise it will ignore it, to meet your codec or `webm` requirement.
> 
> I will add an option to override the codec preference and download format 616 when it exists (thanks for providing a link to 1080p EB video, I use it for testing).

Seems like this isn't working as it should anymore, the output shows it skips 616, but it downloads a regular 1080p version.


[youtube] Extracting URL: https://www.youtube.com/watch?v=VJ2a9wPkGBE
[youtube] VJ2a9wPkGBE: Downloading webpage
[youtube] VJ2a9wPkGBE: Downloading tv client config
[youtube] VJ2a9wPkGBE: Downloading tv player API JSON
[youtube] VJ2a9wPkGBE: Downloading ios player API JSON
[youtube] VJ2a9wPkGBE: Downloading m3u8 information
[info] Testing format 616
ERROR: 
[download] Got error: HTTP Error 403: Forbidden
ERROR: fragment 1 not found, unable to continue
[info] Unable to download format 616. Skipping...
[info] VJ2a9wPkGBE: Downloading 1 format(s): 399+140
[download] Destination: C:\Users\COMBO\Downloads\Resident Evil 3 1999 - Classic Cinematic AI Trailer (Speculative Fiction).f399.mp4
[download] Destination: C:\Users\COMBO\Downloads\Resident Evil 3 1999 - Classic Cinematic AI Trailer (Speculative Fiction).f140.m4a
[Merger] Merging formats into "C:\Users\COMBO\Downloads\Resident Evil 3 1999 - Classic Cinematic AI Trailer (Speculative Fiction).mp4"
Deleting original file C:\Users\COMBO\Downloads\Resident Evil 3 1999 - Classic Cinematic AI Trailer (Speculative Fiction).f399.mp4 (pass -k to keep)
Deleting original file C:\Users\COMBO\Downloads\Resident Evil 3 1999 - Classic Cinematic AI Trailer (Speculative Fiction).f140.m4a (pass -k to keep)

[GUI] yt-dlp.exe process has exited
